go1.x/cookiecutter-aws-sam-hello-golang/{{cookiecutter.project_name}}/README.md

```diff
@@ -17,31 +17,21 @@ This is a sample template for {{ cookiecutter.project_name }} - Below is a brief
 * AWS CLI already configured with Administrator permission
 * [Docker installed](https://www.docker.com/community-edition)
 * [Golang](https://golang.org)
+* SAM CLI - [Install the SAM CLI](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html)
 
 ## Setup process
 
-### Installing dependencies
+### Installing dependencies & building the target 
 
-In this example we use the built-in `go get` and the only dependency we need is AWS Lambda Go SDK:
+In this example we use the built-in `sam build` to automatically download all the dependencies and package our build target.   
+Read more about [SAM Build here](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-cli-command-reference-sam-build.html) 
 
+The `sam build` command is wrapped inside of the `Makefile`. To execute this simply run
+ 
 ```shell
-go get -u github.com/aws/aws-lambda-go/...
+make
```

Comment:
This should be `make build` right? Or will make just auto run the target if there is only one?

Reply:
it will auto-run as it's the only one